font_manager.py

```diff
@@ -0,0 +1,17 @@
+from pygame import font as _font
+from sys import platform as _platform
+
+class FontManager(object):
+    def __init__(self):
+        self._fonts = {}
+        if _platform == "darwin":
```

Comment:
Is there a free font we can use instead? If so, we could avoid system fonts entirely by bundling one. I'm mostly worried about the ubuntu dependency for linux.